hphp/hack/src/hh_single_type_check.ml

```diff
@@ -533,6 +535,14 @@ let parse_options () =
       ( "--disallow-discarded-nullable-awaitables",
         Arg.Set disallow_discarded_nullable_awaitables,
         "Error on using discarded nullable awaitables" );
+      ( "--disable-xhp-element-mangling",
+        Arg.Set disable_xhp_element_mangling,
+        "Disable mangling of XHP elements. With the mangling enabled the new style xhp class will not resolve due to naming mismatch."
```

Comment:
"new style" isn't as helpful for new maintainers. How about something like this?

```
Disable mangling of XHP elements :foo. This is necessary for classes using xhp class modifiers.
```